axelor-project/src/main/java/com/axelor/apps/project/web/SprintController.java

```diff
@@ -171,8 +171,7 @@ public void showAllSprints(ActionRequest request, ActionResponse response) {
     ActionView.ActionViewBuilder actionViewBuilder = ActionView.define(I18n.get("Sprints"));
     actionViewBuilder.model(Sprint.class.getName());
     actionViewBuilder.add("grid", "sprint-grid");
```

Comment:
Call another grid view that doesn't have the toolbar, can't create and can't remove please